yarn.lock

```diff
@@ -4262,7 +4262,7 @@ __metadata:
     ethers: ^5.4.6
     nock: 13.2.9
     sinon: 12.0.1
-    starknet: 5.8.0
+    starknet: 5.24.3
```

Comment:
You should actually use the latest `6.x` beta since it supports newer RPC versions

Reply:
Looks like v6 might be out https://github.com/starknet-io/starknet.js/commit/260e77ac0e8670cc2b22eda8067854c7bc3970b5

Reply:
@archseer  sounds good! I've upgraded the package to 6.0.0-beta.15 - just a heads up there were a few build errors that I ran into once I did this

The latest 6.x beta version depends on [this](https://www.npmjs.com/package/abi-wan-kanabi) package, and if the code is compiled via `yarn setup` with the `noUnusedLocals` and `noUnusedParameters` options set to true, then typescript throws some errors (see screenshot below)

For now I've disabled these options in `/packages/sources/layer2-sequencer-health/tsconfig.json` and left a `TODO` to remove them once the compile errors are resolved. However, if this is not ideal, I'm happy to use a different approach to resolve the issue just lmk

![Screenshot 2024-02-08 at 2 50 57 PM](https://github.com/smartcontractkit/external-adapters-js/assets/147140544/9a747fe4-2ed3-460e-acfa-6b52cdd8fca0)